lib/puppetlabs_spec_helper/module_spec_helper.rb

```diff
@@ -20,35 +20,42 @@ def verify_contents(subject, title, expected_lines)
 module_path = File.join(fixture_path, 'modules')
 
 module_path = [module_path, env_module_path].join(File::PATH_SEPARATOR) if env_module_path
+
 if ENV['SIMPLECOV'] == 'yes'
```

Comment:
Is that if statement really required when you later on already catch the LoadError?

Reply:
This is a way to force it to be fatal. That way you can detect misconfigurations (missing in Gemfile). You could also argue you don't want coverage all the time (because it's slower, distracts you, etc). If you want to change it, I'd move that to a separate discussion and merge this now. It's now neatly contained to just making codecov optional.